src/test/java/seedu/guest/model/util/SampleDataUtilTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.guest.model.util;
+
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.guest.model.GuestBook;
+import seedu.guest.model.guest.DateRange;
+import seedu.guest.model.guest.Email;
+import seedu.guest.model.guest.Guest;
+import seedu.guest.model.guest.Name;
+import seedu.guest.model.guest.NumberOfGuests;
+import seedu.guest.model.guest.Phone;
+
+public class SampleDataUtilTest {
+
+    @Test
+    public void getSampleGuests() {
+        // New Guest[] -> return false
+        assertNotEquals(SampleDataUtil.getSampleGuests(),
+                new Guest[]{
+                    new Guest(new Name("Alex Yooh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
```

Comment:
Could we make these tests clearer by abstracting the changed guest array into an attribute of the class?
Also, would it be appropriate to make the difference more obvious by using either completely different (placeholder) values, or to use an empty array?
Should we also do equals and non-null tests for these methods?

Reply:
Made difference more obvious and added empty array and null test cases.